RecastDemo compiling error 'statement may fall through'

Linux / g++ 7.1.1

Got this error during compilation RecastDemo: 

> **Source/MeshLoaderObj.cpp:101:5: error: this statement may fall through [-Werror=implicit-fallthrough=]**

This error [described here](https://dzone.com/articles/implicit-fallthrough-in-gcc-7)

I use an additional `-Wimplicit-fallthrough=0` flag at RecastDemo.make / ALL_CXXFLAGS to solve this problem.

Detailed log:
```
> premake5 gmake; cd Build/gmake; make

==== Building DebugUtils (debug) ====
==== Building Detour (debug) ====
==== Building DetourCrowd (debug) ====
==== Building DetourTileCache (debug) ====
==== Building Recast (debug) ====
==== Building RecastDemo (debug) ====
MeshLoaderObj.cpp
../../Source/MeshLoaderObj.cpp: In function ‘char* parseRow(char*, char*, char*, int)’:
../../Source/MeshLoaderObj.cpp:101:5: error: this statement may fall through [-Werror=implicit-fallthrough=]
     if (start) break;
     ^~
../../Source/MeshLoaderObj.cpp:102:4: note: here
    default:
    ^~~~~~~
cc1plus: all warnings being treated as errors
make[1]: *** [RecastDemo.make:192: obj/Debug/RecastDemo/MeshLoaderObj.o] Error 1
make: *** [Makefile:69: RecastDemo] Error 2
```

--- a/RecastDemo/Source/MeshLoaderObj.cpp
+++ b/RecastDemo/Source/MeshLoaderObj.cpp
@@ -99,6 +99,7 @@ static char* parseRow(char* buf, char* bufEnd, char* row, int len)
 			case '\t':
 			case ' ':
 				if (start) break;
+				// else falls through
 			default:
 				start = false;
 				row[n++] = c;

